### ISSUE TITLE
Acknowledgement of DB Browser for SQLite developers?

### ISSUE THREAD

**Issue by justinclift** [Appsmata/vSongBook4PC#37]
Hiya,

Just received an email notification from Travis about a failed build here, and took a look through the code repository.

Interesting project. :wink:

As a general thought, would you be ok to add some kind of acknowledgement that substantial pieces of the code have come from the DB Browser for SQLite project?  eg in the History section or something?  That's the standard way of doing things when borrowing code from other Open Source projects. :smile:

**Comment by JacksiroKe** [comment]
I am surprised that you could burg into my repo and accuse me of not acknowledge me of this. At least you could be kind enough to tell me why i cant build to travis and snapcraft. Until then acknowledgement is the least of my concern at the moment

**Comment by justinclift** [comment]
> I am surprised that you could burg into my repo and accuse me of not acknowledge me your code.

Huh?  Sorry, I looked through the History part of your project and didn't see any mention of us, though you've clearly copied parts of [our code](https://github.com/sqlitebrowser/sqlitebrowser/).

Note, there's **nothing wrong** with copying the code of other Open Source projects.  Assuming the licenses are compatible, which they are here.  That being the case, it's good practise to acknowledge the source projects, if for good will if nothing else. :wink:

Like yourself, we've copied code from others to "make things work" when needed.  It's pretty standard. :wink:

With the problems you've mentioned regarding Travis and Snapcraft... I'm not sure what those problem are.  The alerts I received from Travis seemed to be because you'd left our email addresses in the initial Travis configuration you copied over.  So, when you hit build issues while integrating stuff, we got the error emails too (maybe instead of you?).

Anyway, seems like the Travis config isn't set up to alert us any more, so it won't be emailing error stuff to us any more. :+1:

With the Travis problem, I **think** @MKleusberg set our Travis stuff originally.  He might be able to give pointers in getting yours working, but I'm not 100% certain.  It's not an area of things I've touched much before personally.

**Comment by justinclift** [comment]
For the Snap stuff, @deepsidhu1313 has been doing ours.  He **may** be willing to give pointers with yours too, depending on his free time (etc). :smile:

**Comment by deepsidhu1313** [comment]
Not an expert but i will be happy to help on weekends. :)

**Comment by JacksiroKe** [comment]
Look, I had done the acknowledgement thing but when I failed to build on snapcraft I reverted some changes. Somehow my Readme was affected with old stuff from my other project I had written using wxWidghets.

For your information I was not even hitting build on travis at all. I was focused in building on snapcraft for the past one week. I tried your whole entire project and it did build successfully on snapcraft. I didn't know it did that on travis too. So after that I began to eliminate the parts of the projects I didn't need since my main concern has been to build something working on windows and linux. Building a deb package proved impossible but snapcraft didn't. 

If i manage to build on snapcraft I will do all that documentation and acknowledgement and move on to other projects
